### ISSUE TITLE
Getting a non-existing key on Android and in the browser

### ISSUE THREAD

**Issue by martinsotirov** [martinkasa/capacitor-secure-storage-plugin#2]
When I try to get an non-existing or removed key on iOS the promise correctly gets rejected but on Android and in the browser it resolves with a weird looking value:

```
{"value":"\u009eée"}
```

I can implement a check in the resolve callback as a workaround but this might be something worth looking into.

**Comment by nate-g** [comment]
This value comes from calling `atob(null)`, at least on the browser. I'm thinking it's probably something similar on Android. If I get time I'll create a PR, though I'm not currently setup to test on Android.

**Comment by Nodonisko** [comment]
For me it is also returning some strange values on iOS like `eé`

**Comment by dennisameling** [comment]
We use the following workaround for now where we check for `atob(null)`:

```
return SecureStoragePlugin.get({ key }).then(val => {
            if (val.value && val.value !== atob(null)) {
                this.setValue(val.value);
            }
}).catch(error => {
            console.log('Error occured');
});
```

Will provide a PR at least for the web part when I have time.

**Comment by Mageek627** [comment]
I lost some time because of this issue. I think it would be a good idea to specify that in the documentation of `SecureStoragePlugin.get` in the meantime, until it gets fixed.

**Comment by martinkasa** [comment]
Should be fixed in v0.4.0. To use this version you have to update your app to use Capacitor v2. I added getPlatform() method, so you can check if really native plugin is used and not just web "fallback".
